lldb/include/lldb/lldb-enumerations.h

```diff
@@ -1321,10 +1321,11 @@ enum CompletionType {
   eTypeCategoryNameCompletion = (1ul << 24),
   eCustomCompletion = (1ul << 25),
   eThreadIDCompletion = (1ul << 26),
+  eManagedPluginCompletion = (1ul << 27),
   // This last enum element is just for input validation.
   // Add new completions before this element,
   // and then increment eTerminatorCompletion's shift value
-  eTerminatorCompletion = (1ul << 27)
+  eTerminatorCompletion = (1ul << 28)
```

Comment:
This eTerminatorCompletion is not good as it will change and any older binaries that linked against a previous version of LLDB could now get `eManagedPluginCompletion` instead of `eTerminatorCompletion`. Can we remove the `eTerminatorCompletion`? It can cause API issues.